examples/sgd/sgd.go

```diff
@@ -18,11 +20,8 @@ func main() {
 		loss.Backward()
 		opt.Step()
 
-		loss.Close()
-		pre.Close()
-		b.Close()
+		runtime.GC()
+		runtime.GC()
```

Comment:
Here, we must call `runtime.GC()` twice.

Here is the log of calling once:

```log
0
1
Close Tensor, 10, 100
Close Tensor, 10, 100
2
Close Tensor, 10, 100
3
Close Tensor, 10, 100
4
Close Tensor, 10, 100
5
6
Close Tensor, 10, 100
Close Tensor, 10, 100
7
Close Tensor, 10, 100
8
Close Tensor, 10, 100
9
Close Tensor, 10, 100
Close Tensor, 100, 10
```

Here is the log of calling twice:

```log
0
Close Tensor, 10, 100
1
Close Tensor, 10, 100
2
Close Tensor, 10, 100
3
Close Tensor, 10, 100
4
Close Tensor, 10, 100
5
Close Tensor, 10, 100
6
Close Tensor, 10, 100
7
Close Tensor, 10, 100
8
Close Tensor, 10, 100
9
Close Tensor, 10, 100
Close Tensor, 100, 10
```

Reply:
Good catch. It seems that we can keep once in the program but keep it in our mind that double-call clears cleanly.

Reply:
I'm not sure. 

Let's imagine a case, a training program runs in a GPU for many batches and occupies all the GPU memory.  But at some time, the GC does not release GPU memory in time, the training program may be crashed due to OOM.